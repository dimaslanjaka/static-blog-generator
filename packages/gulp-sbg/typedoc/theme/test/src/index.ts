export abstract class Example1 {
  /**
   * Wait for the boolean to return
   * @returns True if it waited
   */
  abstract waitForOk: () => Promise<boolean>

  /**
   * Something
   * ```javascript
   * const str = 'hello world';
   * ```
   *
   * @packageDocumentation
   */
  abstract nothing: () => void

  /**
   * Something
   *
   * ```javascript
   * const str = 'hello world';
   * ```
   *
   * @packageDocumentation
   */
  abstract nothingElse: () => void
}
