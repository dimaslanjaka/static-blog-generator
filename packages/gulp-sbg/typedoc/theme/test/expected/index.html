<!doctype html>
<html class="default no-js">
<head>
	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<title>typedoc-neo-theme-test</title>
	<meta name="description" content="">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<link rel="stylesheet" href="assets/css/main.css">
</head>
<body>
<header>
	<div class="tsd-page-toolbar">
		<div class="container">
			<div class="table-wrap">
				<div class="table-cell" id="tsd-search" data-index="assets/js/search.json" data-base=".">
					<div class="field">
						<label for="tsd-search-field" class="tsd-widget search no-caption">Search</label>
						<input id="tsd-search-field" type="text" />
					</div>
					<ul class="results">
						<li class="state loading">Preparing search index...</li>
						<li class="state failure">The search index is not available</li>
					</ul>
					<ul class="results-priority" style="display:none">
						<li data-name="&quot;index&quot;.Example1.waitForOk" data-subtitle="Main method"></li>
					</ul>
					<a href="index.html" class="title">typedoc-neo-theme-test</a>
					&emsp;<a href="https://github.com/google/typedoc-neo-theme" class="title">Theme</a>
				</div>
				<div class="table-cell" id="tsd-widgets">
					<div id="tsd-filter">
						<a href="#" class="tsd-widget options no-caption" data-toggle="options">Options</a>
						<div class="tsd-filter-group">
							<div class="tsd-select" id="tsd-filter-visibility">
								<span class="tsd-select-label">All</span>
								<ul class="tsd-select-list">
									<li data-value="public">Public</li>
									<li data-value="protected">Public/Protected</li>
									<li data-value="private" class="selected">All</li>
								</ul>
							</div>
							<input type="checkbox" id="tsd-filter-inherited" checked />
							<label class="tsd-widget" for="tsd-filter-inherited">Inherited</label>
						</div>
					</div>
					<a href="#" class="tsd-widget menu no-caption" data-toggle="menu">Menu</a>
				</div>
			</div>
		</div>
	</div>
	<div class="tsd-page-title">
		<div class="container">
			<ul class="tsd-breadcrumb">
			</ul>
		</div>
	</div>
</header>
<div class="container container-main">
	<div class="row">
		<div class="col-3 col-menu menu-sticky-wrap menu-highlight">
			<nav class="tsd-navigation primary">
				<ul>
					<li class="globals  ">
						<a href="globals.html"><em>Globals</em></a>
					</li>
					<li class=" tsd-kind-module">
						<a href="modules/_index_.html">index</a>
					</li>
				</ul>
			</nav>
		</div>
		<div class="col-7 offset-3 col-content">
			<h1>typedoc-neo-theme-test</h1>
			<div class="tsd-panel tsd-typography">
				<p>This is an example project for integration and regression testing.</p>
			</div>
			<!--{&quot;options&quot;:&quot;typedoc.json&quot;,&quot;tsconfig&quot;:&quot;/usr/local/google/home/fleker/samples/typedoc-neo-theme/test&quot;,&quot;inputFiles&quot;:[&quot;/usr/local/google/home/fleker/samples/typedoc-neo-theme/test/src/index.ts&quot;],&quot;mode&quot;:1,&quot;includeDeclarations&quot;:false,&quot;entryPoint&quot;:&quot;&quot;,&quot;exclude&quot;:[],&quot;externalPattern&quot;:[],&quot;excludeExternals&quot;:true,&quot;excludeNotExported&quot;:true,&quot;excludeNotDocumented&quot;:false,&quot;excludePrivate&quot;:true,&quot;excludeProtected&quot;:false,&quot;ignoreCompilerErrors&quot;:true,&quot;disableSources&quot;:false,&quot;includes&quot;:&quot;&quot;,&quot;media&quot;:&quot;&quot;,&quot;out&quot;:&quot;docs&quot;,&quot;json&quot;:&quot;&quot;,&quot;theme&quot;:&quot;node_modules/typedoc-neo-theme/bin/default&quot;,&quot;name&quot;:&quot;&quot;,&quot;includeVersion&quot;:false,&quot;excludeTags&quot;:[],&quot;readme&quot;:&quot;&quot;,&quot;defaultCategory&quot;:&quot;Other&quot;,&quot;categoryOrder&quot;:[],&quot;categorizeByGroup&quot;:true,&quot;gitRevision&quot;:&quot;&quot;,&quot;gitRemote&quot;:&quot;origin&quot;,&quot;gaID&quot;:&quot;&quot;,&quot;gaSite&quot;:&quot;auto&quot;,&quot;hideGenerator&quot;:false,&quot;toc&quot;:[],&quot;disableOutputCheck&quot;:true,&quot;help&quot;:false,&quot;version&quot;:false,&quot;plugin&quot;:[],&quot;logger&quot;:&quot;console&quot;,&quot;listInvalidSymbolLinks&quot;:false,&quot;links&quot;:[{&quot;label&quot;:&quot;Theme&quot;,&quot;url&quot;:&quot;https://github.com/google/typedoc-neo-theme&quot;}],&quot;outline&quot;:[],&quot;search&quot;:[{&quot;name&quot;:&quot;\&quot;index\&quot;.Example1.waitForOk&quot;,&quot;subtitle&quot;:&quot;Main method&quot;}],&quot;source&quot;:[{&quot;path&quot;:&quot;https://github.com/google/typedoc-neo-theme/blob/master/test/src/&quot;,&quot;line&quot;:&quot;L&quot;}]}-->
		</div>
		<div class="col-2 col-menu secondary-menu">
			<nav class="tsd-navigation secondary menu-sticky">
				<ul class="before-current">
				</ul>
			</nav>
		</div>
	</div>
</div>
<footer class="with-border-bottom">
</footer>
<div class="container tsd-generator">
	<p>Generated using <a href="http://typedoc.org/" target="_blank">TypeDoc</a></p>
</div>
<div class="overlay"></div>
<script src="assets/js/main.js"></script>
</body>
</html>